docs/en/stack/ml/anomaly-detection/ml-limitations.asciidoc

```diff
@@ -235,6 +235,16 @@ When you create an {anomaly-job}, you cannot use a field with the
 `data_description` object. This limitation applies irrespective of whether you
 create jobs in {kib} or by using APIs.
 
+[float]
+[[ml-limitations-runtime]]
+== {anomaly-detect-cap} performs better on indexed fields
+
+{anomaly-jobs-cap} sort all data by a user defined time field which is accessed 
+frequently. If the time field is a {ref}/runtime.html[runtime field], the small 
+performance overhead of the runtime field may become significant which slows 
+down the job. Please consider using an indexed field as a time field.
```

Comment:
```suggestion
{anomaly-jobs-cap} sort all data by a user-defined time field, which is frequently 
accessed. If the time field is a {ref}/runtime.html[runtime field], the 
performance impact of calculating field values at query time can significantly slow
the job. Use an indexed field as a time field when running anomaly detection jobs.
```

Reply:
Made some clarifying suggestions, namely to indicate that calculating the values at query time is what might slow jobs. Also suggest stronger wording to "Use an indexed field" rather than considering it.